microverse-repo2/index.html

```diff
@@ -0,0 +1,49 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title>Product Landing Page</title>
+
+    <style>
+@media (max-width:500px) {
+    nav{
+      height:100%;
+      overflow:hidden;
+      display: flex;
+      align-items: center;
+      text-align: center;
+      flex-direction: column;
+    }
+  }

```

Comment:
Required changes #3: For the best experience, you need to use the stylesheet in a separate file. 👍